answer.js

```diff
@@ -0,0 +1,57 @@
+//Question 1
+function sumOfThree (num1,num2,num3) {
+    return num1+num2+num3
+  }
+
+//Question 2
+function colorCar(x) {
+    return `a ${x} car`
+}
+
+//Question 3
+let person = {firstName:"John", lastName:"Doe", age:50, eyeColor:"blue"}
+
+function objectPrinter(person) {
+  return (person)
```

Comment:
You can also use a for...of loop to print out the properties and values from this object.
See [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of)